chrome/injecttop.browserify.js

```diff
@@ -0,0 +1,49 @@
+window.browserpassDisplayTOTP = function(login) {
+  if(login.digits) {
+    var totpBox = document.createElement("div");
+
+    totpBox.style = `
+      position: fixed;
+      top: 0;
+      right: 0;
+      background-color: white;
+      padding: 8px;
+      border-bottom-left-radius: 4px;
+      border-left: 1px solid #888888;
+      border-bottom: 1px solid #888888;
+      font-family: monospace;
+      z-index: 1000000;
+    `;
+
+    var totpText = document.createElement("span");
+    totpBox.appendChild(totpText);
+
+    totpText.innerText = "TOTP:";
```

Comment:
Previously it was `(login.label || "OTP") + ":"`, I think you should preserve that 🙂